install_config/install/disconnected_install.adoc

```diff
@@ -0,0 +1,550 @@
+
+[[disconnected-syncing-repos]]
+=== Syncing Repositories
+
+[NOTE]
+====
+Before you sync with the required repositories, you may need to import the appropriate GPG key:
++
+----
+rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
+----
++
+If the key is not imported, the indicated package is deleted after syncing the repository. 
+====
+
+. Register the server with Red Hat’s customer portal. You will need to use
+the login and password associated with the account that has access to the
+OpenShift subscriptions.
++
+----
+# subscription-manager register
+----
+
+. Attach to a subscription that provides OpenShift channels. You can find the list of available subscriptions using: 
++
+----
+# subscription-manager list --available
+----
++
+Then, find the pool ID for the subscription that provides OpenShift, and attach it:
++
+----
+# subscription-manager attach --pool=${pool_id}
+# subscription-manager repos --disable="*"
+# subscription-manager repos \
+    --enable="rhel-7-server-rpms" \
+    --enable="rhel-7-server-extras-rpms" \
+    --enable="rhel-7-server-ose-3.1-rpms"
+----
+
+. The `yum-utils` command provides *reposync*, which lets you mirror yum repositories, and `createrepo` can create a usable yum repository from a directory.
++
+----
+# yum -y install yum-utils createrepo docker git
+----
++
+You will need up to 110GB of free space in order to sync the software.
+Depending on how restrictive your organization’s policies are, you could
+re-connect this server to the disconnected LAN and use it as the repository
+server. You could use USB-connected storage and transport the software to
+another server that will act as the repository server. This topic covers these options.
+
+. Make a path to where you want to sync the software (either locally or on your
+USB or other device):
++
+----
+# mkdir -p /path/to/repos
+----
+
+. Sync the packages and create the repo for each of them. Note you will need
+to modify the command for the appropriate path you created above.
++
+----
+# for repo in \
+rhel-7-server-rpms rhel-7-server-extras-rpms \
+rhel-7-server-ose-3.1-rpms
+do
+  reposync --gpgcheck -lm --repoid=${repo} --download_path=/path/to/repos
+  createrepo -v /path/to/repos/${repo}
+done
+----
+
+[[disconnected-syncing-images]]
+=== Syncing Images
+
+. Start the Docker daemon:
++
+----
+# systemctl start docker
+----
+
+. Pull all of the required OpenShift dockerized components:
++
+----
+# docker pull registry.access.redhat.com/openshift3/ose-haproxy-router:v3.1.0.4
+# docker pull registry.access.redhat.com/openshift3/ose-deployer:v3.1.0.4
+# docker pull registry.access.redhat.com/openshift3/ose-sti-builder:v3.1.0.4
+# docker pull registry.access.redhat.com/openshift3/ose-docker-builder:v3.1.0.4
+# docker pull registry.access.redhat.com/openshift3/ose-pod:v3.1.0.4
+# docker pull registry.access.redhat.com/openshift3/ose-docker-registry:v3.1.0.4
+----
```

Comment:
We can probably restructure this as:

    # for image in \
        ose-haproxy-router:v3.1.0.4 \
        ose-deployer:v3.1.0.4 \
        ose-sti-builder:v3.1.0.4 \
        ose-docker-builder:v3.1.0.4 \
        ose-pod:v3.1.0.4 \
        ose-docker-registry:v3.1.0.4
    do
      docker pull registry.access.redhat.com/openshift3/${image}
    done